perception/tracking_object_merger/config/decorative_tracker_merger.param.yaml

```
@@ -22,5 +22,5 @@
       max_dt: 1.0
 
     # logging settings
-    enable_logging: false
-    log_file_path: "/tmp/decorative_tracker_merger.log"
+    enable_logging: true
```

Thanks for reworking.
This packages logging information is too detailed, so I do not set this True now.

```suggestion
    enable_logging: false
```